ws/src/main/scala/de/thm/ii/fbs/controller/LoginController.scala

```diff
@@ -66,6 +66,12 @@ class LoginController extends CasClientConfigurerAdapter {
           co.setHttpOnly(false)
           co.setMaxAge(30)
           response.addCookie(co)
+          val cr = new Cookie("JSESSIONID", "")
+          cr.setPath("/")
+          cr.setHttpOnly(true)
+          cr.setSecure(true)
+          cr.setMaxAge(0)
```

Comment:
What happens if MaxAge is set to 0? Will it even work or will it be always invalid?

Reply:
A [MaxAge](https://docs.oracle.com/javaee/6/api/javax/servlet/http/Cookie.html#setMaxAge(int)) of zero deletes the Cookie.